textattack/commands/attack/attack_args_helpers.py

```diff
@@ -414,7 +414,8 @@ def parse_dataset_from_args(args):
 def parse_logger_from_args(args):
     # Create logger
     attack_log_manager = textattack.loggers.AttackLogManager()
-    out_time = int(time.time() * 1000)  # Output file
+    # Get current time for file naming
+    year, month, day, hour, minute = map(int, time.strftime("%Y %m %d %H %M").split())
```

Comment:
Why do you split then convert to ints when we convert it back to a string later. Just format the string how you want it here?